.planning/phases/01-windows-vm-provisioning/01-01-PLAN.md

```
@@ -0,0 +1,189 @@
+    - `grep "ssh-public-key" delete_resources.yaml` returns a match (this task remains)
+    - `grep "sysprep" delete_resources.yaml` returns a match (this task remains)
+    - `grep "user-data" delete_resources.yaml` returns a match (this task remains)
+    - `grep "Display deletion status" delete_resources.yaml` returns a match (final task remains)
+    - The file is valid YAML: `python3 -c "import yaml; yaml.safe_load(open('collections/ansible_collections/osac/templates/roles/ocp_virt_vm/tasks/delete_resources.yaml').read())"`
+  </acceptance_criteria>
+  <done>Orphaned "Delete cloud-init secret" task removed; all other delete tasks intact; file is valid YAML; create/delete symmetry holds for all remaining resources</done>
+</task>
+
+## Verification
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)